### PR TITLE
New contact form

### PR DESCRIPTION
**Pull request by dextmorgn**
Created new contact form backend, with Zapier

### CODE REVIEW

File: gatsby/on-render-body.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ const onRenderBody = ({ setPostBodyComponents }) => {
       `,
       }}
     />,
+    <script type="text/javascript" id="hs-script-loader" async defer src="//js-na1.hs-scripts.com/20445448.js"></script>
```
Comment (AlexHff) [inline gatsby/on-render-body.js]: delete

File: package.json [hunk 2]
```diff
@@ -36,9 +36,11 @@
   "dependencies": {
     "@sentry/gatsby": "^6.5.1",
     "@sentry/tracing": "^6.5.1",
+    "axios": "^0.21.1",
     "bluebird": "^3.7.2",
     "classnames": "^2.3.1",
     "codecov": "^3.8.2",
+    "cors": "^2.8.5",
```
Comment (AlexHff) [inline package.json]: delete

File: src/components/ContactForm/ContactForm.js [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+  const [contactCheck, setContactCheck] = useState('');
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const contact = {
+      name, firstname, phone, email, message
+    };
+    axios
+      .post(
+        'https://hooks.zapier.com/hooks/catch/10583025/b28wqv3/silent/',
```
Comment (AlexHff) [inline src/components/ContactForm/ContactForm.js]: Move to .env

File: src/templates/page-template.js [hunk 4]
```diff
@@ -5,26 +5,36 @@ import Layout from '../components/Layout';
...
-    <Layout title={`${pageTitle} - ${siteTitle}`} description={metaDescription} socialImage={socialImageUrl} >
+    <Layout
+      title={`${pageTitle} - ${siteTitle}`}
+      description={metaDescription}
+      socialImage={socialImageUrl}
+    >
       <Page title={pageTitle}>
         <div dangerouslySetInnerHTML={{ __html: pageBody }} />
       </Page>
+      {pageTitle === 'Contact me' && <ContactForm />}
```
Comment (AlexHff) [inline src/templates/page-template.js]: Contact as page title

File: src/templates/page-template.js [hunk 5]
```diff
@@ -5,26 +5,36 @@ import Layout from '../components/Layout';
...
 
   return (
-    <Layout title={`${pageTitle} - ${siteTitle}`} description={metaDescription} socialImage={socialImageUrl} >
+    <Layout
+      title={`${pageTitle} - ${siteTitle}`}
+      description={metaDescription}
+      socialImage={socialImageUrl}
+    >
       <Page title={pageTitle}>
         <div dangerouslySetInnerHTML={{ __html: pageBody }} />
```
Comment (AlexHff) [inline src/templates/page-template.js:31]: Try to move ContactForm here

**Review by AlexHff — CHANGES_REQUESTED** [review]

File: src/components/ContactForm/ContactForm.js [hunk 6]
```diff
@@ -12,22 +15,22 @@ const ContactForm = () => {
     'En validant, vous autorisez HOFMANN.AI à stocker les données personnelles soumises afin qu\'elles fournissent le contenu souhaité.'
   );
   const [contactCheck, setContactCheck] = useState('');
-
+  if (url === null || url === undefined) return <></>;
```
Comment (AlexHff) [inline src/components/ContactForm/ContactForm.js:18]: Should be `if (!url) return <></>;`

**Review by AlexHff — CHANGES_REQUESTED** [review]

File: src/components/ContactForm/ContactForm.js (hunk 6 above)
Comment (dextmorgn) [reply src/components/ContactForm/ContactForm.js:18]: We tester it out with Jacques, and it did not pass the linter.

**Review by AlexHff — APPROVED** [review]

**Review by AlexHff — APPROVED** [review]

**Pull request merged**